docs/sources/_index.md

```diff
@@ -39,3 +39,5 @@ If it's your first time using the guide, start with the [Get started](https://gr
 
 Writers' Toolkit is open source and available at [`grafana/writers-toolkit`](https://github.com/grafana/writers-toolkit).
 If you have questions, or feedback on how to improve this documentation, [open an issue](https://github.com/grafana/writers-toolkit/issues/new) and help make this an even better resource.
+
+Test deploy preview
```

Comment:
I have not dug into the code yet but at first glance I am not seeing this change on the deploy preview.

Reply:
🤦 
Of course! I don't actually check out the PR branch. I need to fix that in the script...

Reply:
It's there now: https://deploy-preview-writers-toolkit-1014-zb444pucvq-vp.a.run.app/docs/writers-toolkit/#:~:text=even%20better%20resource.-,Test%20deploy%20preview,-Was%20this%20page